nature/models.py

```diff
@@ -287,15 +261,18 @@ class Observation(ProtectedNatureModel):
     occurrence = models.ForeignKey(Occurrence, models.PROTECT, db_column='esiintymaid', blank=True, null=True)
     created_time = models.DateTimeField(db_column='pvm_luotu')
     last_modified_time = models.DateTimeField(blank=True, null=True, db_column='pvm_editoitu')
+    protection_level = models.ForeignKey('ProtectionLevel', models.PROTECT, db_column='suojaustasoid')
 
     class Meta:
         managed = False
         ordering = ['id']
         db_table = 'lajihavainto'
 
+    def __str__(self):
+        return self.id  # TODO: need to find more meaningful string representation
 
-class Species(ProtectedNatureModel):
-    id = models.IntegerField(primary_key=True)
+
+class Species(models.Model):
     taxon = models.CharField(max_length=5, blank=True, null=True, db_column='ryhma')
```

Comment:
Is this field name correct? seems the original column name means something different?

Reply:
Well, taxon is a general name of any level of the taxonomical hierarchy, that's why I used it here. It seems this column refers to the highest order taxon in the system, which is https://en.wikipedia.org/wiki/Kingdom_(biology)  (because the strings here are "animal", "plant" or "fungus"), but because it is called "ryhmä" in Finnish, i don't know if this is reserved for other levels of taxon as well. That's why I used the general term "taxon" :D

Reply:
Oh, in general I tried to look for the correct names in the Finnish Species Registry API documentation at https://apitest.laji.fi/explorer/#/Information 

However, our system is considerably simpler, and also they might not have all the same fields, so some of our names might well be wrong or different to those used by the Finnish Species Registry.

Reply:
Thanks for clarification. I think we can keep as it is, you surely have better Finnish than me :smile:  Just get a bit confused as the following two fields are `taxon_1` and `taxon_2` and they use different Finnish words elioryhma1 and elioryhma2

Reply:
Yeah I think here ryhma is just short for elioryhma. These are apparently all meant for storing taxa of different levels, so the first `taxon` contains the highest level name, and `taxon_1` and `taxon_2` contains lower and still lower level names.